python/tests/unit/connectors/ollama/services/test_ollama_chat_completion.py

```diff
@@ -6,94 +6,187 @@
 
 from semantic_kernel.connectors.ai.ollama.ollama_prompt_execution_settings import OllamaChatPromptExecutionSettings
 from semantic_kernel.connectors.ai.ollama.services.ollama_chat_completion import OllamaChatCompletion
-from semantic_kernel.contents.chat_history import ChatHistory
-from tests.unit.connectors.ollama.utils import MockResponse
+from semantic_kernel.exceptions.service_exceptions import ServiceInitializationError
 
 
-def test_settings():
-    ollama = OllamaChatCompletion(ai_model_id="test_model")
+def test_settings(model_id):
+    ollama = OllamaChatCompletion(ai_model_id=model_id)
     settings = ollama.get_prompt_execution_settings_class()
     assert settings == OllamaChatPromptExecutionSettings
 
 
+def test_init_empty_service_id(model_id):
+    ollama = OllamaChatCompletion(ai_model_id=model_id)
+    assert ollama.service_id == model_id
+
+
+@pytest.mark.parametrize("exclude_list", [["OLLAMA_MODEL"]], indirect=True)
+def test_init_empty_model_id(ollama_unit_test_env):
+    with pytest.raises(ServiceInitializationError):
+        _ = OllamaChatCompletion(env_file_path="fake_env_file_path.env")
+
+
+@pytest.mark.asyncio
+@patch("ollama.AsyncClient.__init__", return_value=None)  # mock_client
+@patch("ollama.AsyncClient.chat")  # mock_chat_client
+async def test_custom_host(
+    mock_chat_client,
+    mock_client,
+    model_id,
+    service_id,
+    host,
+    chat_history,
+    prompt,
+    default_options,
+):
+    mock_chat_client.return_value = {"message": {"content": "test_response"}}
+
+    ollama = OllamaChatCompletion(ai_model_id=model_id, host=host)
+    _ = await ollama.get_chat_message_contents(
```

Comment:
Would be good to also check if the parsing is done correctly

Reply:
What do you mean by parsing?